src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,7 +1,20 @@
-// Don't forget to import the React library
+import React from 'react';
+import { User } from '../../types';
 
-// Create a `UserInfo` component accepting a `user` object and use it to render
-// a `todo.user` in the list with some styling. (Show at least a `name` and an
-// `email` of the `user`)
+type Props = {
+  user: User;
+};
 
-// Add a default export statement for UserInfo component to use it in the other files
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const {
+    name,
+    email,
+  } = user;
+
+  return (
```

Comment:
`Props` complicate your code - use type `User` directly
```suggestion
export const UserInfo: React.FC<User> = ({ name, email }) => (
```